File: C#/TurkeyHolidayValidator.cs
using System;
using HolidayChecker;

namespace HolidayChecker.Turkey
{
    /// <summary>
    /// TurkeyHolidayValidator
    ///
    /// This class is used to determine if a given date is a holiday in Turkey.
    /// </summary>
    public class TurkeyHolidayValidator : HolidayValidator
    {
        public TurkeyHolidayValidator(bool includeSaturday) : base(includeSaturday)
        {
        }
    }
}

File: Java/TurkeyHolidayValidator.java
package HolidayChecker.Turkey;

import HolidayChecker.HolidayValidator;

/**
 * TurkeyHolidayValidator
 *
 * This class is used to determine if a given date is a holiday in Turkey.
 */
public class TurkeyHolidayValidator extends HolidayValidator {
    public TurkeyHolidayValidator(boolean includeSaturday) {
        super(includeSaturday);
    }
}
